tests/gnrc_tcp/tests/05-garbage-pkts.py

```diff
@@ -96,14 +97,24 @@ def test_short_header(child, src_if, src_ll, dst_if, dst_l2, dst_ll, dst_port):
 @testfunc
 def test_send_ack_instead_of_syn(child, src_if, src_ll,
                                  dst_if, dst_l2, dst_ll, dst_port):
+    # check if any debug output is generated during `@testfunc` decorator 
+    # execution. If so, send fewer packets to prevent the target node
+    # from being overwhelmed by packets and output.
+    debug = child.expect([pexpect.TIMEOUT,
+                          r'GNRC_TCP: Enter "\S+", File: .+(\d+)\s'], timeout=1)
```

Comment:
Ah yes, parenthesis are special characters in python regexes ;-P

```suggestion
                          r'GNRC_TCP: Enter "\S+", File: .+\(\d+\)\s'], timeout=1)
```

Reply:
Always nice to work with regex :D